docs/micropython.md

```diff
@@ -0,0 +1,49 @@
+# Running on MicroPython
+
+The module is designed for [MicroPython]. You can run it on embedded systems,
+and, thanks to the [MicroPython UNIX and Windows port], also on Linux, macOS,
+and Windows.
+
+While MicroPython on a workstation does not provide hardware support usually
+available when running MicroPython on embedded devices, this is mostly not
+a concern when running a database driver.
+
+
+## Embedded
+
+Todo.
```

Comment:
Just a nit.

Well, this section may absorb a few words about how to whip the example programs onto a device, both for vanilla MicroPython, and CircuitPython, eventually also using a separate documentation file for the latter.

I mean, as long as this project bears just or primarily a README, it will be perfectly fine to keep that in [README » Installation](https://github.com/crate/micropython-cratedb#installation), so maybe just refer to that section until the project will gain a dedicated documentation slot, rendered on RTD for example, where `docs/` will need to be polished further.